index.js

```diff
@@ -5,23 +5,7 @@ import packageJson from './package.json' with {type: 'json'};
 
 const deprecatedRules = createDeprecatedRules({
 	// {ruleId: ReplacementRuleId | ReplacementRuleId[]}, if no replacement, use `{ruleId: []}`
-	'import-index': [],
-	'no-array-instanceof': 'unicorn/no-instanceof-array',
-	'no-fn-reference-in-iterator': 'unicorn/no-array-callback-reference',
-	'no-reduce': 'unicorn/no-array-reduce',
-	'no-unsafe-regex': [],
-	'prefer-dataset': 'unicorn/prefer-dom-node-dataset',
-	'prefer-event-key': 'unicorn/prefer-keyboard-event-key',
-	'prefer-exponentiation-operator': 'prefer-exponentiation-operator',
-	'prefer-flat-map': 'unicorn/prefer-array-flat-map',
-	'prefer-node-append': 'unicorn/prefer-dom-node-append',
-	'prefer-node-remove': 'unicorn/prefer-dom-node-remove',
-	'prefer-object-has-own': 'prefer-object-has-own',
-	'prefer-replace-all': 'unicorn/prefer-string-replace-all',
-	'prefer-starts-ends-with': 'unicorn/prefer-string-starts-ends-with',
-	'prefer-text-content': 'unicorn/prefer-dom-node-text-content',
-	'prefer-trim-start-end': 'unicorn/prefer-string-trim-start-end',
-	'regex-shorthand': 'unicorn/better-regex',
```

Comment:
Those rules deprecated long enough to remove.

Reply:
Agreed